.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -19,14 +19,16 @@ A clear and concise description of what you expected to happen.
 If applicable, add screenshots and/or console outputs to help explain your problem.
 
 **Info (please complete the following information):**
+Stakeholders and components can be a comma separated list or on multiple lines.
+If you add a new stakeholder or component, not on the sample list, add it on a line by its own. 
```

Comment:
People will open this in text-edit mode and then maybe remove it from the final submission, so it will  be in plain text anyway.
So I'd prefer to leave it in regular text (a NOTE will disturb more who's reading the ticket (if left there) than help people writing it). Anyway I don't feel strongly either way. So if you feel it's important I'll add NOTE. Let me know.

Reply:
I agree with what you said. This comment can be ignored then. Approving the PR now!